### PR TITLE
Fix codec error in md

### PR DESCRIPTION
**Pull request by tarikcurto**
## Changes
- replaced curly quotes by straight quotes. 

### Linked issues
curly quotes creates a codification error in windows machines.
image

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -54,7 +54,7 @@
 
     DISCLAIMER; LIMITATION OF LIABILITY. 
 
-    THE LICENSED MATERIALS ARE PROVIDED “AS-IS” AND WITH ALL FAULTS. 
+    THE LICENSED MATERIALS ARE PROVIDED "AS-IS" AND WITH ALL FAULTS.
```
Comment (JCZuurmond) [inline LICENSE:57]: @tarikcurto : Could you revert this change?

File: README.md [hunk 2]
```diff
@@ -1423,13 +1423,13 @@ Admin privileges required for commands:
 
 **Python:** UCX needs Python version 3.10 or later.
 
-**Solution:** Check the current version using “python --version”. If the
+**Solution:** Check the current version using "python --version". If the
```
Comment (JCZuurmond) [inline README.md:1426]: Now we are at it 

```suggestion
**Solution:** Check the current version using `python --version`. If the
```

File: README.md [hunk 3]
```diff
@@ -1423,13 +1423,13 @@ Admin privileges required for commands:
...
 version is lower than 3.10, upgrade the local Python version to 3.10 or
 higher.
 
 **Databricks CLI:** Databricks CLI v0.213 or higher is needed.
 
-**Solution:** Check the current version with “databricks --version”. For
+**Solution:** Check the current version with "databricks --version". For
```
Comment (JCZuurmond) [inline README.md:1432]: Same here

```suggestion
**Solution:** Check the current version with `databricks --version`. For
```

File: README.md [hunk 4]
```diff
@@ -1443,8 +1443,8 @@ UCX version.
 **Solution:** Upgrade UCX, and rerun the assessment job before running
 the migration workflows. For some reason, if you want to install a
 specific version of UCX, you can do it using the command
-“databricks labs install ucx@\<version\>”, for example,
-“databricks labs install ucx@v0.21.0”.
+"databricks labs install ucx@\<version\>", for example,
+"databricks labs install ucx@v0.21.0".
```
Comment (JCZuurmond) [inline README.md:1447]: And here:

```suggestion
`databricks labs install ucx@\<version\>`, for example,
`databricks labs install ucx@v0.21.0`.
```

File: README.md [hunk 5]
```diff
@@ -1482,17 +1482,17 @@ workspace.
 
 **Solution:** The Databricks CLI provides an option to select the
 [<u>profile</u>](https://docs.databricks.com/en/dev-tools/cli/profiles.html)
-using “*--profile \<profile_name\>*” or “*-p \<profile_name\>*”. You can
+using "*--profile \<profile_name\>*" or "*-p \<profile_name\>*". You can
```
Comment (JCZuurmond) [inline README.md:1485]: And here:

```suggestion
using `--profile \<profile_name\>` or `-p \<profile_name\>`. You can
```

File: README.md [hunk 6]
```diff
@@ -1482,17 +1482,17 @@ workspace.
...
 test that the correct workspace is getting selected by running any
-Databricks CLI command. For example, you can run “*databricks clusters
-list -p prod*” and check that the Prod clusters are being returned. Once
+Databricks CLI command. For example, you can run "*databricks clusters
+list -p prod*" and check that the Prod clusters are being returned. Once
```
Comment (JCZuurmond) [inline README.md:1488]: and here

```suggestion
Databricks CLI command. For example, you can run `databricks clusters
list -p prod` and check that the Prod clusters are being returned. Once
```

File: README.md [hunk 7]
```diff
@@ -1482,17 +1482,17 @@ workspace.
...
 the profile is verified, you can run UCX install for that specific
-profile: “*databricks labs install ucx -p prod*”.
+profile: "*databricks labs install ucx -p prod*".
```
Comment (JCZuurmond) [inline README.md:1490]: and here:

```suggestion
profile: `databricks labs install ucx -p prod`.
```

File: README.md [hunk 8]
```diff
@@ -1482,17 +1482,17 @@ workspace.
...
 
 **Account Level:** Multiple account level profiles are set in the
 .databrickscfg file.
 
-**Solution:** The installation command “*databricks labs install ucx*”
+**Solution:** The installation command "*databricks labs install ucx*"
```
Comment (JCZuurmond) [inline README.md:1495]: and here

```suggestion
**Solution:** The installation command `databricks labs install ucx`
```

File: README.md [hunk 9]
```diff
@@ -1529,7 +1529,7 @@ under Data Access Configuration of SQL Warehouse Admin Settings.
 
 ### Verify the Installation
 
-Once the UCX command “*databricks labs install ucx*” has completed
+Once the UCX command "*databricks labs install ucx*" has completed
```
Comment (JCZuurmond) [inline README.md:1532]: and here:

```suggestion
Once the UCX command `databricks labs install ucx` has completed
```

**Review by JCZuurmond — CHANGES_REQUESTED** [review]
@tarikcurto : Thank you for your contribution! Generally it looks good, I added some suggestions:

**Comment by tarikcurto** [comment]
@JCZuurmond thank you so much for your improvements, accepted all them and  reverted license quote change.

**Review by JCZuurmond — APPROVED** [review]
Great, thanks for your contribution and fixing this issue for us 🚀 !

**Comment by nfx** [comment]
@tarikcurto please reply with "I have acknowledged and signing this CLA - https://gist.github.com/smikailli/25902b59426b533154bd985b571dde70" in order for us to be able to merge this

**Comment by tarikcurto** [comment]
> I have acknowledged and signing this CLA - https://gist.github.com/smikailli/25902b59426b533154bd985b571dde70"

I have acknowledged and signing this CLA - https://gist.github.com/smikailli/25902b59426b533154bd985b571dde70"
